sdcm/cluster.py

```diff
@@ -2322,7 +2322,7 @@ def update_scylla_packages(node, queue):
             node.remoter.run('sudo yum update -y --skip-broken')
             # replace the packages
             node.remoter.run('yum list installed | grep scylla')
-            node.remoter.run('sudo rpm -URvh --replacefiles /tmp/scylla/* | true')
+            node.remoter.run('sudo rpm -URvh --replacefiles /tmp/scylla/*', ignore_status=True, verbose=True)
```

Comment:
rethinking about it, why do we ignore status?
why did we have the "true" on first place...
I would expect that if the rpm install fails, the test will fail.

Reply:
yeah, seem like that if we are using this feature, it's important to use that this one should work.
yeah we should change that

Reply:
@fgelcer please change `ignore_status` to `False`

Reply:
@bentsi , done